src/main/scala/nl.knaw.dans.easy.bagstore/server/BagsServletComponent.scala

```diff
@@ -74,21 +75,22 @@ trait BagsServletComponent extends DebugEnhancedLogging {
       val uuidStr = params("uuid")
       (multiParams("splat") match {
         case Seq(path) =>
-          ItemId.fromString(s"""$uuidStr/${ path }""")
+          ItemId
+            .fromString(s"""$uuidStr/${ path }""")
             .recoverWith {
               case _: IllegalArgumentException => Failure(new IllegalArgumentException(s"invalid UUID string: $uuidStr"))
-            }
-            .flatMap(itemId => {
-              debug(s"Retrieving item $itemId")
-              bagStores.copyToStream(itemId, request.header("Accept").flatMap(acceptToArchiveStreamType) , response.outputStream)
-            })
+            }.flatMap(itemId => {
```

Comment:
not sure if you changed something here, but can you put the `.flatMap(...` on a new line? This causes an unnecessary diff as well as weird formatting. As far as I can see here, you didn't change anything, right?